### DISCUSSION TITLE
Failed to create Doc

### DISCUSSION THREAD

**Discussion by BrydonP** [PaloAltoNetworks/docusaurus-openapi-docs#385] [General]
Hi All,

Some of my pages are failing to create because of the error "Error: Operation must have summary or operationId defined" any idea on how this is happening, I don't see why some pages would developed fine and other fail to do so.

Anyone have any ideas?

**Comment by sserrata** [comment]
As the error states, each path/operation requires either a `summary` or `operationId`. It's also permitted to have both.

**Reply by BrydonP** [reply]
Thanks, I figured it out after posting, but I wasn't able to find a way to delete the question until it was too late! I appreciate the response!
